nixos/modules/virtualisation/digital-ocean-config.nix

```diff
@@ -0,0 +1,165 @@
+{ config, pkgs, lib, modulesPath, ... }:
+with lib;
+{
+  imports = [
+    (modulesPath + "/profiles/qemu-guest.nix")
+    (modulesPath + "/virtualisation/digital-ocean-init.nix")
+  ];
+  options.virtualisation.digitalOcean = with types; {
+    setRootPassword = mkOption {
+      type = bool;
+      default = true;
```

Comment:
I'm considering changing this default to `false` because:
1. It's a pretty insecure option, setting the password to something accessible from the metadata server to any unauthenticated user on the system that can access the network.
2. It currently resets the root password on every boot rather than first boot. I should maybe add a path condition to make sure it only ever runs once?
3. DO always tells me my custom images don't support root password setting, so I can't even get it to email me the password to test this. Maybe I'm using the wrong Distribution setting when uploading it?